### ISSUE TITLE
Revocation check should be implemented

### ISSUE THREAD

**Issue by mlegner** [admin-ch/CovidCertificate-App-Android#49]
https://github.com/admin-ch/CovidCertificate-App-Android/blob/060a40c7510c113a304ebe515e0f7833ae03623b/eval/src/main/java/ch/admin/bag/covidcertificate/eval/Eval.kt#L84-L86

I suspect that the revocation check should do more than just return `SUCCESS`.

**Comment by mlegner** [comment]
This issue needs to be moved to the [CovidCertificate-SDK-Android](https://github.com/admin-ch/CovidCertificate-SDK-Android) repository after #48 and https://github.com/admin-ch/CovidCertificate-SDK-Android/pull/1 are merged.

**Comment by goebelUB** [comment]
Indeed, this is currently a placeholder as long as the revocation check is unimplemented.
This feature requires support from the backend to compile the revocation list and make it available to the clients.
Both the backend and the client teams are working on this.
